src/search/landmarks/landmark_count_heuristic.cc

```diff
@@ -263,20 +254,48 @@ bool LandmarkCountHeuristic::generate_helpful_actions(
 }
 
 bool LandmarkCountHeuristic::landmark_is_interesting(
-    const State &state, const LandmarkNodeSet &reached, LandmarkNode &lm_node) const {
-    /* A landmark is interesting if it hasn't been reached before and
-     its parents have all been reached, or if all landmarks have been
-     reached before, the LM is a goal, and it's not true at moment */
-
-    int num_reached = reached.size();
-    if (num_reached != lgraph->get_num_landmarks()) {
-        if (reached.find(&lm_node) != reached.end())
-            return false;
-        else
-            return !check_node_orders_disobeyed(lm_node, reached);
+    const State &/*state*/, LandmarkNode &lm_node) const {
+    /*
+      A landmark is interesting if it hasn't been reached before and
+      its parents have all been reached, or if all landmarks have been
+      reached before, the LM is a goal, and it's not true at moment.
```

Comment:
```suggestion
      reached before, the LM is a goal, and it's not true at the moment.
```

Reply:
Looks like Jendrik's comment is not yet resolved.